static/src/javascripts/lib/fetch-json.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import config from 'lib/config';
+import fetch from 'lib/fetch';
+
+function json(input: string, init: any = {}) {
+    // #? we should use Object.assign for this assignment, but this currently breaks
+    // Karma tests that depend on fetch-json.js and have not been stubbed
+    const options = init;
```

Comment:
Functionally equivalent yes. I have structured it like this for 2 reasons:

1. [mutating the `init` object](https://github.com/guardian/frontend/blob/60bce285302cd92a4e845acafe7905dec0bf5827/static/src/javascripts/lib/fetch-json.js#L13) directly causes an ESLint error
2. Reassigning in this way acts as a reminder to do it properly using `Object.assign` later